shell/run_shell_cmd.go

```diff
@@ -230,23 +230,29 @@ type CmdOutput struct {
 	Stderr string
 }
 
+var gitTopLevelDir string = ""
```

Comment:
Not sure if this approach will work in case of multiple git repositories

Reply:
Good point, I'll consider this